voc/python/klass.py

```diff
@@ -51,6 +58,10 @@ def __init__(
         self._constructor = None
 
     @property
+    def full_name(self):
+        return '.'.join(self.namespace.split('.')[1:] + [self.name])
```

Comment:
The complication here will be what Java can actually support. Java has major problems when you have a class and a namespace with the same name. If there are capitalisation changes, you're generally OK, but having a `MyClass1.class` and `MyClass1/InnerClass.class` in the same project will be a problem.

Based on that, I suspect the best option will be to use the `<module>` namespace, but use a classname of `MyClass1$InnerClass` (which is Java's general naming approach for "anonymous" or "inner" pieces).